### ISSUE TITLE
List file size on files tab and sum them when downloading

### ISSUE THREAD

**Issue by inodb** [ncihtan/htan-portal#154]
Would be nice to see file size on the files tab

- [ ] Check if there is a file size column in the JSON
    - [ ] Add it if not
- [ ] Add filesize column on files tab
- [ ] Show total download size of selected files somewhere (either in popup or on selection)

**Comment by alisman** [comment]
@inodb doesn't seem like there's any file size data in current json

**Comment by inodb** [comment]
yes we should add it in as part of the pull from synapse. Eg. hdash does the same: http://htan_dashboard.surge.sh/ & https://github.com/ncihtan/hdash/
